packages/bsky/src/views/index.ts

```diff
@@ -216,6 +227,16 @@ export class Views {
     }
   }
 
+  knownFollowers(
+    knownFollowers: Required<HydratorProfileViewerState>['knownFollowers'],
+    state: HydrationState,
+  ) {
+    const followers = mapDefined(knownFollowers.followers, (did) => {
+      return this.profileBasic(did, state)
+    })
+    return { count: knownFollowers.count, followers }
```

Comment:
I believe that there is a minimum of one item for `followers`.  It would be nice to have some logic enforcing that colocated around here, or perhaps drop the requirement from the lexicon if you think the frontend wont mind.

We may also want to consider enforcing blocks, and possibly mutes, in this list.

Reply:
[We updated](https://github.com/bluesky-social/atproto/pull/2557#discussion_r1631762637) this the other day to `0`. Yeah I'll make sure frontend can handle it, no issue there.

Yeah should def filter out blocks, I think mutes are OK to leave in though.

Reply:
Ya know, hadn't codegenned though, glad you brought this up.

Blocks are now filtered, added a test